source/modulo_components/modulo_components/lifecycle_component.py

```diff
@@ -87,6 +87,9 @@ def __handle_configure(self) -> bool:
         """
         try:
             result = self.on_configure_callback()
+            if result is None:
+                self.get_logger().error("'on_configure_callback' doesn't return a value, transition will be rejected.")
```

Comment:
If you agree with my earlier comment, then this applies to all of the following error texts